tests/test_devices.py

```diff
@@ -255,6 +255,152 @@ def test_device_inventory_update_success(client, mocker):
     assert device._inventory == inventory
 
 
+class test_inputs:
+    def test_property_populated(client, mocker):
+        """Should check if the device property are correctly populated"""
+        device = AlarmDevice(client)
+        mocker.spy(device._connection, "query")
+        device.connect("username", "password")
+        inputs = {
+            0: "Entryway Sensor",
+            1: "Outdoor Sensor 1",
+            2: "Outdoor Sensor 2",
+        }
+        # Test
+        device.update()
+        device_inputs = {}
+        for input_id, name in device.inputs:
+            device_inputs.update({input_id: name})
+        assert device_inputs == inputs
+
+    def test_all_properties_empty(client, mocker):
+        """Should check store the e-connect System status in the inventory device object."""
```

Comment:
I would recommend to explain better what this test tests. Something along the lines: "Ensure the property returns an empty dict if `_inventory` is empty".